exercises/src/main/scala/fpinscala/errorhandling/Either.scala

```diff
@@ -4,34 +4,57 @@ package fpinscala.errorhandling
 import scala.{Option => _, Either => _, Left => _, Right => _, _} // hide std library `Option` and `Either`, since we are writing our own in this chapter
 
 sealed trait Either[+E,+A] {
- def map[B](f: A => B): Either[E, B] = sys.error("todo")
-
- def flatMap[EE >: E, B](f: A => Either[EE, B]): Either[EE, B] = sys.error("todo")
-
- def orElse[EE >: E, B >: A](b: => Either[EE, B]): Either[EE, B] = sys.error("todo")
-
- def map2[EE >: E, B, C](b: Either[EE, B])(f: (A, B) => C): Either[EE, C] = sys.error("todo")
+ def map[B](f: A => B): Either[E, B] = this match {
+		case Left(e) => Left(e)
```

Comment:
Hey Ashkan, your editor seems to be inserting actual tab characters as indentation. For future reference, could you set it to expand tabs into 2 spaces? We are uniformly using 2 spaces for indentation everywhere.